### DISCUSSION TITLE
Installer for 1Remotte

### DISCUSSION THREAD

**Discussion by STrRedWolf** [1Remote/1Remote#388] [General]
**Is your enhancement related to a problem? Please describe.**
1Remote's missing an installer

**Describe the solution you'd like**
Use the free InnoSetup program to build an installer.

**Describe alternatives you've considered**
(None, used to InnoSetup in my dayjob)

**Additional context**
May provide a proper InnoSetup Script file for the build.

**Comment by majkinetor** [comment]
Why would you need one when you can simply unpack and run, or install via windows store or winget or chocolatey pacakge manager ?

**Comment by STrRedWolf** [comment]
- Unpack & Run: That's what I'm doing now but it's missing from my Start menu
- Install via Windows Store:  Really? I have more trouble getting that to do the right thing (just install it, don't spam my desktop, don't put junk all over the place, and don't give me any grief about it).  No, give me an MSI that does the job.  Simple.
- WinGet:  I need to install one thing to get another? (bad enough the Store already installed)
- Chocolatey Package manager:  Do I need to repeat myself?

Besides, this works but can be better, so the issue is opened so I can have something to check in against.

**Comment by majkinetor** [comment]
I personally can't stand installers, but to each his own.  Winget is already installed btw. `winget install premotem` should do the job. Along with icon and autoupdate. However, soon it wont be free. Icon/start menu can be created without installer, btw, there are number of tools doing that, its even better because if its get deleted you can recreate them from the app.

Maintaining installer is a thing too, so I would rather have that time and effort invested in better features of the main tool 

> don't put junk all over the place, and don't give me any grief about it

Its funny that you dislike junk and want installer. Portable software is as good as it gets regarding this. 1Remote detects first run, so it can provide stuff you want on it anyway without any kind of installer that you need to handle after installation.

**Comment by VShawn** [comment]
I do have a program packaged as an MSIX, but it has never been tested. 

It may be a matter of personal taste, but I also do not like installer. I don't want a installer to mess up my start menu and desktop icons. As for installation, both Chocolatey and WinGet can install the app by one-click, which is why I have only provided portable versions all this time.
